next/components/forms/SelectField/SelectAllDropdownRow.tsx

```diff
@@ -0,0 +1,21 @@
+interface SelectAllDropdownRowProps {
+  divider?: boolean
+  onSelectAll: () => void
+}
+
+const SelectAllDropdownRow = ({ divider, onSelectAll }: SelectAllDropdownRowProps) => {
+  return (
+    <div className="flex h-14 w-full cursor-pointer flex-col bg-white px-5 hover:text-form-black-hover" onClick={onSelectAll}>
+      <div className="flex h-full flex-col justify-center">
+        <div className="flex flex-row justify-center">
+          <p className="w-full text-p-md font-semibold">
+            Select all
```

Comment:
please pull this into the `public/locales/[lang]/forms.json` translation files and use `useLocale` hook's `t` function